pkgs/top-level/all-packages.nix

```diff
@@ -15589,7 +15587,7 @@ with pkgs;
       else if platform.isAndroid then 12
       else if platform.isLinux then 11
       else if platform.isWasm then 12
-      else latest_version;
+      else 14;
```

Comment:
This seems like (imo) the one place where picking the latest version might actually be what we want? (assuming that newest LLVM is our best bet at providing support for other/untested platforms)

On the other hand there probably aren't many users relying on this fallback anyways (even if we forget to bump this it's probably okay) and I'd rather see us gain the ability to [specify the LLVM version as part of the system config](https://github.com/NixOS/nixpkgs/commit/52f870793098fae92a701d44eff77b423ca080cc) anyways. So I think this is fine.